perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/FacilityAttributeProcessor.java

```diff
@@ -70,14 +75,14 @@ public void processAllAttributesRemoved(String msg, MessageBeans beans) {
 		Perun perun = ldapcManager.getPerunBl();
 
 		// ensure we have the correct beans available
-		if(beans.getUser() == null) {
+		if (beans.getUser() == null) {
```

Comment:
This is probably original copy/paste bug. We should check if Facility is present within the beans and not User, since we delete all Facility attributes here.